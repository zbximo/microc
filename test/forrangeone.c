void main()
{
    Int i;
    Int j;
    j = 1;
    for i in range(3) {
        print i;
    }

}