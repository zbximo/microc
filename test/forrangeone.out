24 19 0 5 25 15 1 15 1 13 0 1 1 0 1 12 15 -1 13 0 0 1 0 0 12 15 -1 16 54 13 0 0 1 11 22 15 -1 15 0 13 0 0 1 13 0 0 1 11 0 1 1 12 15 -1 13 0 0 1 11 0 3 7 18 29 15 -2 21 -1